src/iam.ts

```diff
@@ -74,7 +72,8 @@ export type TestIamPermissionsResponse = [object[], r.Response];
  * @param {object} apiResponse The full API response.
  */
 export interface TestIamPermissionsCallback {
-  (err?: Error|null, permissions?: object|null, apiResponse?: r.Response): void;
+  (err?: Error|null, permissions?: {[key: string]: boolean}|null,
```

Comment:
The `{[key: string]: boolean}` type feels like something we could export and re-use, no?